src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1555,7 +1544,7 @@ protected override void SerializeCore(XmlElement element, SaveContext context)
             helper.SetAttribute("isVisible", IsVisible);
             helper.SetAttribute("isUpstreamVisible", IsUpstreamVisible);
             helper.SetAttribute("lacing", ArgumentLacing.ToString());
-            helper.SetAttribute("isSelectedInput", IsSelectedInput.ToString());
+            helper.SetAttribute("isSelectedInput", IsInputNode.ToString());
```

Comment:
is this left here so reach continues working? could we put a comment documenting why the other string is used?